xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -526,7 +534,7 @@
       </Parameters>
       <Docs>
         <param name="item">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Adds an item to the <see cref="T:System.Collections.Generic.ICollection`1" />.</summary>
```

Comment:
Same. Needs to be updated to this type:
```suggestion
        <summary>Adds an item to the <see cref="T:System.Text.RegularExpressions.CaptureCollection" />.</summary>
```

Reply:
Another choice is to simply use natural language. Looks a bit nicer. The user already knows they are visiting the current type, so there's no point in indicating it over and over. Example:

```suggestion
        <summary>Adds an item to the collection.</summary>
```

Reply:
I like the natural language better. I'm going to go with that.